index.html

```diff
@@ -998,8 +993,13 @@ <h2>Presentations</h2>
 Credentials MAY be used to make a <a>verifiable presentation</a>. A
 verifiable presentation may be of a verifiable credential itself, or
 of data derived from verifiable credentials in a format that is
-cryptographically verifiable. In the example below, we show a
-verifiable presentation that embeds verifiable credentials:
+cryptographically verifiable. Some schemes support claims specifying
+the date of birth of a subject to be used to prove that its age lies in
+an interval or is greater or less than a some value which is needed in use
+cases like proving drinking age or teenager discount or elderly discount;
+other schemes might be more simple and require the holder to share the credential.
+The holder has the flexibility to use the claim in any way that is applicable to that
+claim. In the example below, we show a verifiable presentation that embeds verifiable credentials:
```

Comment:
Suggest wording change:

"Some schemes may enable holders to prove mathematical statements about a claim in a credential without revealing the claim itself, such as proving that the date of birth of a subject lies in an interval or is greater or less than some value. This provides a more privacy enhanced mechanism for use cases such as proving drinking age or teenager discount or elderly discount when other personally identifiable information such as email address or photo ID is not also required. Other schemes might be more simple and require the holder to share the credential and its claims directly. The holder has the flexibility..."

Reply:
Changed

Reply:
Hmm, I'm not seeing the change here. I wonder if github diff is being inconsistent.

Reply:
Added back